app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+ protect_from_forgery with: :exception
+ before_action :create_cart
```

Comment:
Why do we need to create a shopping cart for every controller in our application?